js/custom.js

```diff
@@ -0,0 +1,402 @@
+
+/*====================================
+ SIDEBAR MENU SCRIPTS
+ ======================================*/
+
+/* Set the width of the side navigation to 0 and the left margin of the page content to 0, and the background color of body to white */
+function closeNav() {
+
+
+    document.getElementById("main").style.marginLeft = "0";
+
+}
+
+
+
+$(document).ready(function(){
+
+    $(".button-menu").click(function(){
+        $(".name").addClass('move-name');
+        $(".lm_bg").addClass('lm_bg_active');
+        $("ul").removeClass('sidebar-hide');
+        $("ul").addClass('sidebar-active');
+        $("ul a:nth-child(2)").removeClass('menu-1')
+        $("ul a:nth-child(2)").addClass('menu-1-transition');
+        $("ul a:nth-child(3)").removeClass('menu-2');
+        $("ul a:nth-child(3)").addClass('menu-2-transition');
+        $("ul a:nth-child(4)").removeClass('menu-3');
+        $("ul a:nth-child(4)").addClass('menu-3-transition');
+        $("ul a:nth-child(5)").removeClass('menu-4');
+        $("ul a:nth-child(5)").addClass('menu-4-transition');
+        $("ul a:nth-child(6)").removeClass('menu-5');
+        $("ul a:nth-child(6)").addClass('menu-5-transition');
+        $("ul a:nth-child(7)").removeClass('menu-6');
+        $("ul a:nth-child(7)").addClass('menu-6-transition');
+
+    });
+    $(".closebtn").click(function(){
+        $(".name").removeClass('move-name');
+        $(".lm_bg").removeClass('lm_bg_active');
+        $("ul").removeClass('sidebar-active');
+        $("ul").addClass('sidebar-hide');
+        $("ul a:nth-child(2)").addClass('menu-1');
+        $("ul a:nth-child(2)").removeClass('menu-1-transition');
+        $("ul a:nth-child(3)").addClass('menu-2');
+        $("ul a:nth-child(3)").removeClass('menu-2-transition');
+        $("ul a:nth-child(4)").addClass('menu-3');

```

Comment:
I saw that all that menu classes are same. Make just one clas e.g. `menu-item` and use it for all menu elements. Then you will not have to use selectors like `ul a:nth-child(4)` because `$('ul a') will catch all those 7 elements`. so `$("ul a").addClass('menu-item');` will add `menu-item` class to all `ul a` elements